src/main/java/dev/pgm/community/sessions/SessionConfig.java

```diff
@@ -0,0 +1,23 @@
+package dev.pgm.community.sessions;
+
+import dev.pgm.community.feature.config.FeatureConfigImpl;
+import org.bukkit.configuration.Configuration;
+
+public class SessionConfig extends FeatureConfigImpl {
+
+  public static final String KEY = "users.sessions";
+
+  public SessionConfig(Configuration config) {
+    super(KEY, config);
+  }
+
+  @Override
+  public void reload(Configuration config) {
+    super.reload(config);
+  }
+
+  @Override
+  protected String getEnabledKey(String key) {
+    return "users.enabled";
```

Comment:
Could we make it so the `users.sessions` is the enable key, or will it not function properly? Alternatively, if we're going to have sessions enable only when users are enabled, then just ditch the `SessionsConfig` and use `UserConfig`. 

If you do end up using `users.sessions`, make sure to add it to the `config.yml` too.